@plotly/dash-component-plugins/src/index.js

```diff
@@ -0,0 +1,3 @@
+import { decorate, isReady } from './dynamicImport';
+
+export { decorate, isReady };
```

Comment:
or just: `export { decorate, isReady } from './dynamicImport';`

I guess the reason to have separate files (rather than putting everything in `index`) is just preemptive structure for future expansion?

Reply:
yes